### ISSUE TITLE
[Dwarven Skirmisher (3 of 3)] 

### ISSUE THREAD

**Issue by eulchen05** [gwentcards/gwentcards.github.io#27]
**Platform**: PC
**Game version**: v 4.04

**Card name**: Dwarven Skirmisher (3 of 3)
**Issue**: site says "Buy from trader" "Oxenfurt, The Alchemy Inn, Stjepan" but I couldn't buy the card there and instead received it as a random card I won

**Comment by vzx** [comment]
Thanks a lot for reporting. Are you sure that you did not buy it from Stjepan before? Did he have the other cards for sale when you visited him? I can change it to include `OR Randomly earned` like some other cards. Various sources were indicating that Stjepan sold the card, which is why I added it to this site.

I would prefer one more report or a double check by someone before changing. I haven't played in some time, perhaps another playthrough of my own is overdue.... :)

**Comment by eulchen05** [comment]
I sadly can't remember which cards I actually bought from Stjepan but I do know that I didn't buy the third Skirmisher from him because I kept looking into his shop for the card but it wasn't there. Eventually I got it from a random NPC win.

I understand your hesitance to change it, just thought I'd let you know since I really like the site and find it very helpful.

**Comment by vzx** [comment]
Yes, you're right. I've changed it now to add `OR Randomly earned` and when I play it again soon I will double check Stjepan to see if I should remove Stjepan as source entirely.

**Comment by CrowdedDread** [comment]
Bought Dwarven Skirmisher (1 of 3) from Stjepan without any of the other in deck.

**Comment by alexislefebvre** [comment]
It may be like “Dol Blathanna Scout” that has this note “Only 1 of each illustration of this card can be acquired. One of the three versions is sold by 2 merchants, but if you already have it, you cannot buy another from the other merchant.” https://witcher.fandom.com/wiki/Dol_Blathanna_Scout_(gwent_card)#Note

I only have “Dwarven Skirmisher” 1 and 2 and I can't buy the 3rd from the “Golden Sturgeon”. Maybe that's not a big deal if the game require only 1 of card of each set? Otherwise I hope that I will earn it randomly.

**Comment by vzx** [comment]
I played the game again (on PC) and didn't buy any cards from Stjepan. Then I went on to collect all other Gwent cards and made a list of what I got, until I didn't get any cards randomly earned anymore. By that time, I only earned 2 Dwarven Skirmisher randomly, no 3rd one. I had to buy the 3rd one from Stjepan.

I am OK leaving it as `Oxenfurt, The Alchemy Inn, Stjepan OR Randomly earned` though, because people will generally just buy it from Stjepan if and when they can. And if it's not there, it gives a hint that apparently in some versions of the game it could be randomly earned.
